crates/dslab-mp/tests/test_mc.rs

```diff
@@ -1138,18 +1157,12 @@ fn permanent_net_problem(#[case] strategy_name: String, net_problem: NetworkProb
                 NetworkProblem::DisableLink => sys.network().disable_link("node1", "node3"),
             }
             sys.send_local_message("process0", Message::new("PING", "some_data"));
-            let mut mc = build_mc(&sys, strategy_name, prune, goal, invariant);
-            mc.run()
+            let mut mc = ModelChecker::new(&sys);
+            run_mc!(mc, strategy_config, strategy_name)
```

Comment:
Можно еще упростить, передавая систему в макрос и создавая ModelChecker уже внутри (кажется в тестах последний нигде не используется несколько раз):

```suggestion
            run_mc!(&sys, strategy_config, strategy_name)
```

Reply:
сделал